periodic/periodic.go

```diff
@@ -128,9 +132,15 @@ type periodicRunner struct {
 	RunnerOptions
 }
 
-// internal version, returning the concrete implementation.
-func newPeriodicRunner(opts *RunnerOptions) *periodicRunner {
-	r := &periodicRunner{*opts} // by default just copy the input params
+var (
+	globalAbort     chan os.Signal
+	outstandingRuns int64
+	abortMutex      sync.Mutex
```

Comment:
gAbortMutex

Or put then all in a structure and call it gSomething

Reply:
k